### PR TITLE
gcp ipi ovn ipsec

### PR DESCRIPTION
**Pull request by anuragthehatter**
@jianlinliu @lihongan

### CODE REVIEW

**Comment by jianlinliu** [comment]
pls squash multiple commits into one.

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/OWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+../OWNERS
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/OWNERS:1]: Suggest to update the OWNERS list to add networking team member there for your own review and approve.

Comment (anuragthehatter) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/OWNERS:1]: @jianlinliu Sure.I can add myself as this feature ipsec is owned by me. So right way to add me would be to add my id in all metadata and OWNERS files?

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/OWNERS:1]: yes

Comment (anuragthehatter) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/OWNERS:1]: That was accidental, sorry

Comment (anuragthehatter) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/OWNERS]: Will need a day to squash and properly fix this. I have two env due to ongoing RH client subscription issue (not able to use docker on local system) and fedora vm has issues syncing up test branches. WIll fix this tomorrow, sorry for inconveneince

**Comment by anuragthehatter** [comment]
Re-opened https://github.com/openshift/release/pull/28533 in lieu of this

**Pull request closed without merging**